README.md

```diff
@@ -110,3 +110,22 @@ By default the jobs will be submitted to HTCondor. If you want to use a differen
       myStudy.submit(1, 5) # 1 stands for sixtrack job, 5 is trial number 
       myStudy.collect_result(1, 5) # 1 stands for sixtrack job, 5 is trial number 
       ```
+
+## Description for the database tables
+
+There are ten tables in the database:
+
+| Table Name | Description |
+|:------------:|:-----------|
+|**boinc\_vars**| store the config parameters for boinc jobs |
```

Comment:
how many lines does this table have? 1 or as many as the BOINC tasks?

Reply:
For the moment, just 1